code/modules/religion/rites/musician_rites.dm

```diff
@@ -0,0 +1,102 @@
+/*********Base**********/
+/datum/religion_rites/song_tuner
+	name = "Tune Song"
+	desc = "this is a base."
+	ritual_length = 10 SECONDS
+	favor_cost = 10
+///if repeats count as continuations instead of a song's end, TRUE
+	var/repeats_okay = TRUE
+///visible message sent to indicate a song will have special properties
+	var/visible_message
+
+/datum/religion_rites/song_tuner/invoke_effect(mob/living/user, obj/structure/altar_of_gods/AOG)
+	user.AddComponent(/datum/component/smooth_tunes, src, repeats_okay)
+
+/**
+  * Perform the song effect.
+  *
+  * Arguments:
+  * * arg1 - Atom (parent) of the smooth_tunes component. This is limited to the compatible items of said component, which currently includes mobs and objects so we'll have to type appropriately.
+  */
+/datum/religion_rites/song_tuner/proc/song_effect(atom/A, datum/song/S)
+
+/*********Subtypes**********/
+
+/datum/religion_rites/song_tuner/evangelism
+	name = "Evangelical Hymn"
+	desc = "Spreads the word of your god, gaining favor for each non-holy listener."
+	favor_cost = 0
+
+/datum/religion_rites/song_tuner/evangelism/song_effect(atom/A, datum/song/S)
+	if(!S || !GLOB.religious_sect)
+		return
+	for(var/i in S.hearing_mobs)
+		if(i == A)
+			continue
+		if(!isliving(i))
+			continue //stinky ghosts
+		var/mob/living/L = i
+		if(L.mind.holy_role)
```

Comment:
you should probably add a mind check somewhere here, so you can't just farm this with NPC monkeys or bees or mice or whatever